rudder-core/src/main/scala/com/normation/rudder/services/policies/write/BuildBundleSequence.scala

```diff
@@ -354,8 +356,7 @@ object CfengineBundleVariables extends AgentFormatBundleVariables {
         val newMain = tb.main match {
           case Nil => Nil
           // in that case, we are using the same ReportId for report and bundle
-          case firstBundle :: tail => (Bundle(firstBundle.id, BundleName(s"""current_technique_report_info("${firstBundle.name.value}")"""), Seq())
-                                       :: firstBundle :: tail)
+          case firstBundle :: tail =>  firstBundle :: tail
```

Comment:
i fail to see what's the benefit of this pattern matching

Reply:
You are totally right, just forgot to clean after removing what need to be removed